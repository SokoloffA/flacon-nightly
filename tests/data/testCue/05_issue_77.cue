PERFORMER "ArtistName"
TITLE "AlbumName"
FILE "01. Track01.flac" WAVE
    TRACK 01 AUDIO
        TITLE "Title01"
        INDEX 01 00:00:00
FILE "02. Track02.flac" WAVE
    TRACK 02 AUDIO
        TITLE "Title02"
        INDEX 01 00:00:00
