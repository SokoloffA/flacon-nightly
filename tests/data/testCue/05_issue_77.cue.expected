[ DISC 01 / TRACK 01 ]
  FILE	      = 01. Track01.flac
  ALBUM       = "AlbumName"
  PERFORMER   = ArtistName
  TITLE       = Title01
  INDEX CD 00 = 00:00:00
  TRACKNUM    = 1
  TRACKCOUNT  = 2
  DISCNUM     = 1
  DISCCOUNT   = 1

[ DISC 01 / TRACK 02 ]
  FILE        = 02. Track02.flac
  ALBUM       = "AlbumName"
  PERFORMER   = ArtistName
  TITLE       = Title02
  INDEX CD 00 = 00:00:00
  TRACKNUM    = 2
  TRACKCOUNT  = 2
  DISCNUM     = 1
  DISCCOUNT   = 1
